api/bertymessenger.proto

```diff
@@ -320,10 +323,12 @@ message Contact {
   Conversation conversation = 3;
   State state = 4;
   string display_name = 5;
+  string avatar_cid = 9  [(gogoproto.moretags) = "gorm:\"column:avatar_cid\"", (gogoproto.customname) = "AvatarCID"];
```

Comment:
```suggestion
  string avatar_cid = 9 [(gogoproto.customname) = "AvatarCID"];
```

sorry, I didn't check, but my current feeling is that the result is the same, nope?

Reply:
nope, it gets called "avatar_c_id" and breaks gorm internal stuff on specific queries

Reply:
created issue upstream https://github.com/go-gorm/gorm/issues/3779